### ISSUE TITLE
[Plugin Library] Report of PclZip not working on alpine/php7

### ISSUE THREAD

**Issue by jstanden** [jstanden/cerb#326]
On #321, it was reported that the PclZip fallback isn't working on the plugin library in alpine Linux with PHP7 in Docker.

The first step should be verifying that tests run against PclZip regardless of whether 'zip' is installed and enabled. It may not be specific to alpine.

**Comment by jstanden** [comment]
See: https://github.com/jstanden/cerb/issues/321#issuecomment-319976721

**Comment by jstanden** [comment]
I see a warning in PclZip with PHP 7.1, but plugins worked fine:

~~~
PHP Deprecated:  Methods with the same name as their class will not be constructors in a future version of PHP; PclZip has a deprecated constructor in ./libs/devblocks/libs/pclzip/pclzip.lib.php on line 190
~~~

We can upgrade the library from Composer.

**Comment by jstanden** [comment]
Fixed in 8.0.5
